src/main/java/com/mraof/minestuck/data/recipe/MinestuckRecipeProvider.java

```diff
@@ -143,6 +145,7 @@ protected void registerRecipes(Consumer<IFinishedRecipe> recipeBuilder)
 		CookingRecipeBuilder.smeltingRecipe(Ingredient.fromItems(MSBlocks.WOODEN_CACTUS), Items.CHARCOAL, 0.15F, 200).addCriterion("has_wooden_cactus", hasItem(MSBlocks.WOODEN_CACTUS)).build(recipeBuilder, new ResourceLocation(Minestuck.MOD_ID, "charcoal_from_wooden_cactus"));
 		
 		cookingRecipesFor(recipeBuilder, Ingredient.fromItems(MSItems.BEEF_SWORD), MSItems.STEAK_SWORD, 0.5F, "has_beef_sword", hasItem(MSItems.BEEF_SWORD));
+		cookingRecipesFor(recipeBuilder, Ingredient.fromItems(MSItems.CAKE_MIX), Blocks.CAKE, 0.0F, "has_cake_mix", hasItem(MSItems.CAKE_MIX));
```

Comment:
This is adding cake recipes under the minecraft domain, meaning that it might compete with the vanilla recipe and recipe advancement for cake. I'd suggest that you instead save those cooking recipes under the minestuck domain